### ISSUE TITLE
Readme on github - role links do not work

### ISSUE THREAD

**Issue by VirusTLNR** [ykundesu/SuperNewRoles#198]
Was looking forward to reading about the roles in your mod but on mobile at least the links are not working.

I thought maybe it's just a linking issue in the English readme but I tried the Japanese readme and had the same issue of the link scrolling up to the top of the readme.

Look forward to reading about your roles etc soon :)

**Comment by satsumaimoamo** [comment]
Thank you for your issue!
I'm sorry. 
We have not yet created an English role description.
We will write English ReadMe soon.

**Comment by VirusTLNR** [comment]
> Thank you for your issue! I'm sorry. We have not yet created an English role description. We will write English ReadMe soon.

it is no worries, i just thought the japanese role info doesnt work too, but now i am on my computer, i see that it is just the english links are broken, no worries, if i get the time i will try to assist using a translator :)

**Comment by satsumaimoamo** [comment]
Thank you!
We will repair English links soon.

**Comment by Kurato-Tsukishiro** [comment]
Sorry for the delay, but the ReadMe has been corrected.

The position descriptions, which were only written in Japanese, have been translated into English.
Also, the link that skipped to the Japanese job title description has been changed to English.

The translation was done using DeepL translation, and we did not check that the text was correct, so there may be some errors. We apologize for any errors.
(In fact, it was pointed out to us in advance that "Impostor" had become "Imposter" in all cases, and we have corrected the mistake.)


Thank you for your input!
